packages/kbn-monaco/src/esql/lib/ast/validation/validation.ts

```diff
@@ -461,22 +464,37 @@ function validateSource(
         locations: source.location,
       })
     );
-  } else if (source.sourceType === 'index' && !sourceExists(source.name, sources)) {
-    messages.push(
-      getMessageFromId({
-        messageId: 'unknownIndex',
-        values: { name: source.name },
-        locations: source.location,
-      })
-    );
-  } else if (source.sourceType === 'policy' && !policies.has(source.name)) {
-    messages.push(
-      getMessageFromId({
-        messageId: 'unknownPolicy',
-        values: { name: source.name },
-        locations: source.location,
-      })
-    );
+  } else {
+    // if it is a wildcard and the command does not support wildcards, then throw
```

Comment:
nit: not really thowing here, right? just showing an error message?

Reply:
Correct, that is just a planning comment before writing the actual code.
I'll remove it.

```suggestion
```